docker-compose.yml

```diff
@@ -271,3 +272,12 @@ services:
       - ./src/prometheus/prometheus-config.yaml:/etc/prometheus/prometheus-config.yaml
     ports:
       - "${PROMETHEUS_SERVICE_PORT}:${PROMETHEUS_SERVICE_PORT}"
+
+  # Grafana
+  grafana:
+    image: grafana/grafana:9.0.1
```

Comment:
Do you know how the ubuntu version of the docker image differs from this one? Maybe this one is based off alpine or something

https://hub.docker.com/r/grafana/grafana/tags?page=1&name=9.0.1

Reply:
According to https://grafana.com/docs/grafana/latest/setup-grafana/installation/docker/

> Each edition is available in two variants: Alpine and Ubuntu.

So `grafana/grafana:9.0.1` is based on Alpine which is smaller than `grafana/grafana:9.0.1-ubuntu`, I think smaller one is more suitable for local development, since it can save more resources.

Reply:
Thanks for finding that info, yup I agree completely!